src/components/PeopleFilters.tsx

```diff
@@ -22,8 +109,9 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={searchQuery}
+            onChange={onChange}
```

Comment:
The `onChange` handler for the search input should update the `searchQuery` state to reflect the current input value. Ensure that the `onChange` function updates the state correctly.